src/content/manifest/hybrid-development.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Hybrid development
+description: Hybrid development allows you to develop and run a service locally while the other components of your app run in the cloud.
+sidebar_label: Hybrid development
+id: hybrid-development
+---
+
+import Image from '@theme/Image';
+
+Okteto offers two different approaches to service development: *sync* ot *hybrid* mode. The command affected by this is the [`okteto up`](/reference/okteto-cli.mdx/#up) command being configurable at manifest level.
+
+The default mode used is the *sync* mode in which changes made to the local service are synchronized with the remote development container deployed in the cluster. On the other hand, the *hybrid* mode allows to run the development service locally and the rest of the application components in the cloud.
+
+### Problem to solve
```

Comment:
```suggestion
### Deciding which mode to use
```

Maybe something like this? "Problem to solve" doesn't seem fitting for public docs since "problem" has a negative connotation. I propose something like this instead 😄

Reply:
3b791f1eb3ff23bf057a17da1065f5533b18133a